frontend/src/react/Exam/DisplayExam.js

```diff
@@ -1,68 +1,78 @@
 import React, {Component} from 'react';
-//import PropTypes from 'prop-types';
+import PropTypes from 'prop-types';
 import {connect} from 'react-redux';
 
-import {getExams} from '../../redux/exams/actions/get';
-import {postExam} from '../../redux/exams/actions/post';
+import {getExam} from '../../redux/exams/actions/getSingle';
+import Error from '../utils/Error';
+import DisplayExercice from './DisplayExercice';
 
 class DisplayExam extends Component {
 
-    static propTypes = {
+    state = {
+        displayError: true
+    };
 
+    static propTypes = {
+        exam: PropTypes.object.isRequired
     };
 
     componentDidMount() {
-        const { id } = this.props.route.match.params;
-        console.log(id);
-        //TODO: get single Exam: this.props.fetchExam();
-    }
+        this.props.fetchExam();
+    };
 
-    /*createExam = () => {
-        this.props.postExam({
-            title: 'This is a title from Redux',
-            'reminders': 'This is a reminder from Redux',
-            'instruction': 'These are some instructions'
-        });
-    }*/
+    closeError = () => {
+        this.setState({ displayError: false });
```

Comment:
Shouldn't we do :
```
this.setState({
...this.state, 
displayError: false 
});
```
?

Reply:
I guess this works, I think I actually used the same syntax before 🙊

Reply:
Yep, fair point. Just didn't have anything else in my state so don't need do, do I?

Reply:
You don't 👍